code/game/machinery/teleporter.dm

```diff
@@ -56,7 +56,7 @@
 				return
 			L.forceMove(src)
 			locked = L
-			to_chat(user, "<span class='caution'>You insert the GPS device into the [src]'s slot.</span>")
+			to_chat(user, "<span class='caution'>You insert the GPS device into [src]'s slot.</span>")
```

Comment:
I think I'll leave this one as it is, since it might end up being too long if I change it.
e.g:
`"You insert the GPS device into the teleporter control console's slot."`
vs
`"You insert the global positioning system (NERD) into the teleporter control console's slot."`